jbehave-core/src/main/java/org/jbehave/core/model/TableTransformerMonitor.java

```diff
@@ -0,0 +1,13 @@
+package org.jbehave.core.model;
+
+import org.jbehave.core.model.ExamplesTable.TableProperties;
+
+/**
+ * Interface to monitor table transformer events
+ */
+public interface TableTransformerMonitor {
+
+    void beforeTransformerApplying(String transformerName, TableProperties properties, String inputTable);
+
+    void afterTransformerApplying(String transformerName, TableProperties properties, String outputTable);
```

Comment:
maybe something like beforeTransformation and afterTransformation?

Reply:
> maybe something like beforeTransformation and afterTransformation?

It seems to me that the current name reflects the subject area more clearly. "beforeTransformation" can be regarded as "after applying all transformers" or in some other way